include/wabt/interp/interp.h

```diff
@@ -588,12 +588,12 @@ struct Value {
     // Alternatively we could modify InterpAlloca to set the type.
     assert(t == type || type == ValueType::Any);
   }
-  ValueType type;
 #else
   Value() : v128_(0, 0, 0, 0) {}
   void SetType(ValueType) {}
   void CheckType(ValueType) const {}
 #endif
+  ValueType type;
```

Comment:
Can you add comment here.  How about `only used in debug builds but declared here to avoid struct size differing`.

Actually though, you know what, I think you might be correct about putting this in `config.h`.  If we have a release version of libwabt we really want to fix this header to be release too.  

Also, the sizeof values in the interpreter is likely very important to performance of the release build (not sure if anyone actually cares about that).

Would it be tricky to put this in config.h and fix it at library build time?